<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-SRR3L3F" failures="8" tests="20" name="Surefire test" time="0.069" errors="0" timestamp="2021-04-25T14:12:28 MSK">
  <testcase classname="PenTest" name="testDosomethingElsePrint" time="0.0"/>
  <testcase classname="PenTest" name="testIsWork" time="0.0"/>
  <testcase classname="PenTest" name="testIsWork" time="0.0">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at PenTest.testIsWork(PenTest.java:58)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 37 stack frames]]>
    </failure>
  </testcase> <!-- testIsWork -->
  <testcase classname="PenTest" name="testIsWork" time="0.0"/>
  <testcase classname="PenTest" name="testIsWork" time="0.006">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at PenTest.testIsWork(PenTest.java:58)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 37 stack frames]]>
    </failure>
  </testcase> <!-- testIsWork -->
  <testcase classname="PenTest" name="testIsWork" time="0.0"/>
  <testcase classname="PenTest" name="testWrite" time="0.0"/>
  <testcase classname="PenTest" name="testWrite" time="0.0"/>
  <testcase classname="PenTest" name="testWrite" time="0.0"/>
  <testcase classname="PenTest" name="testWrite" time="0.0">
    <failure type="java.lang.StringIndexOutOfBoundsException" message="begin 0, end 11, length 4">
      <![CDATA[java.lang.StringIndexOutOfBoundsException: begin 0, end 11, length 4
at java.base/java.lang.String.checkBoundsBeginEnd(String.java:3756)
at java.base/java.lang.String.substring(String.java:1902)
at Pen.write(Pen.java:33)
at PenTest.testWrite(PenTest.java:73)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- testWrite -->
  <testcase classname="PenTest" name="testWrite" time="0.0"/>
  <testcase classname="PenTest" name="testWrite" time="0.0"/>
  <testcase classname="PenTest" name="testWrite" time="0.0"/>
  <testcase classname="PenTest" name="testWrite" time="0.0"/>
  <testcase classname="PenTest" name="testWrite" time="0.0"/>
  <testcase classname="PenTest" name="testWrite" time="0.0">
    <failure type="java.lang.StringIndexOutOfBoundsException" message="begin 0, end 20, length 12">
      <![CDATA[java.lang.StringIndexOutOfBoundsException: begin 0, end 20, length 12
at java.base/java.lang.String.checkBoundsBeginEnd(String.java:3756)
at java.base/java.lang.String.substring(String.java:1902)
at Pen.write(Pen.java:33)
at PenTest.testWrite(PenTest.java:73)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- testWrite -->
  <testcase classname="PenTest" name="testWrite" time="0.0">
    <failure type="java.lang.AssertionError" message="expected [Work] but found [Work ]">
      <![CDATA[java.lang.AssertionError: expected [Work] but found [Work ]
at PenTest.testWrite(PenTest.java:73)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 38 stack frames]]>
    </failure>
  </testcase> <!-- testWrite -->
  <testcase classname="PenTest" name="testWrite" time="0.0">
    <failure type="java.lang.StringIndexOutOfBoundsException" message="begin 0, end 10, length 6">
      <![CDATA[java.lang.StringIndexOutOfBoundsException: begin 0, end 10, length 6
at java.base/java.lang.String.checkBoundsBeginEnd(String.java:3756)
at java.base/java.lang.String.substring(String.java:1902)
at Pen.write(Pen.java:33)
at PenTest.testWrite(PenTest.java:73)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- testWrite -->
  <testcase classname="PenTest" name="testWrite" time="0.0">
    <failure type="java.lang.AssertionError" message="expected [] but found [Work]">
      <![CDATA[java.lang.AssertionError: expected [] but found [Work]
at PenTest.testWrite(PenTest.java:73)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 38 stack frames]]>
    </failure>
  </testcase> <!-- testWrite -->
  <testcase classname="PenTest" name="testgetColor" time="0.0">
    <failure type="java.lang.AssertionError" message="expected [Blue] but found [BLUE]">
      <![CDATA[java.lang.AssertionError: expected [Blue] but found [BLUE]
at PenTest.testgetColor(PenTest.java:67)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 38 stack frames]]>
    </failure>
  </testcase> <!-- testgetColor -->
</testsuite> <!-- Surefire test -->
